pkgs/development/python-modules/pybrowserid/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, requests, mock }:
+
+buildPythonPackage rec {
+  pname = "PyBrowserID";
+  version = "0.14.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qvi79kfb8x9kxkm5lw2mp42hm82cpps1xknmsb5ghkwx1lpc8kc";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  buildInputs = [ mock ];
```

Comment:
Shouldn't this rather be `checkInputs` ?

Reply:
Ahh, yeah.

Reply:
Corrected.